app/views/users/new.html.haml

```diff
@@ -1,16 +1,38 @@
+%script(src="https://js.stripe.com/v2/")
+:javascript
+  Stripe.setPublishableKey("#{ENV['STRIPE_PUBLISHABLE_KEY']}");
+= javascript_include_tag 'payment'
+
 %section.register.container
   .row
     .col-sm-10.col-sm-offset-1
-      = bootstrap_form_for @user, html: {class: "form-horizontal"} do |f|
+      = bootstrap_form_for @user, html: {class: "form-horizontal", id: "payment-form"} do |f|
```

Comment:
`class: "form-horizontal"` is no longer correct. The current version of the gem uses `layout: :horizontal`, as you can see in the docs.